src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SentenceIdentificationService.kt

```diff
@@ -187,16 +204,16 @@ class SentenceIdentificationService {
         releaseDateTypes += TUSED
       }
 
-      if (doesHdcedDateApply(sentence, offender)) {
+      if (doesHdcedDateApply(sentence, offender, sentence.isMadeUpOfOnlyDtos())) {
         releaseDateTypes += HDCED
       }
     }
     return releaseDateTypes
   }
 
-  private fun doesHdcedDateApply(sentence: CalculableSentence, offender: Offender): Boolean {
+  private fun doesHdcedDateApply(sentence: CalculableSentence, offender: Offender, isMadeUpOfOnlyDtos: Boolean): Boolean {
```

Comment:
Do you need to pass in the boolean here? Couldn't you call the sentence.isMadeUpOfOnlyDtos() in this method